Add a chapter to existing blog for filling survey "CoreDNS adoption"

fturib opened the pull request:
This blog was written to present GA of CoreDNS for k8s v1.11

Following this GA, we need to get some feedback from those who are adopting CoreDNS as the DNS server in their Kubernetes clusters.

I though enhancing this blog would be a good way to advertize this survey.


I also fixed a typo.

fturib commented:
/assign @neolit123  - please let me know if updating a blog, is a valid way in this case. Thanks

fturib commented:
@johnbelamaric - FYI.

neolit123 commented on content/en/blog/_posts/2018-07-10-coredns-ga.md:
```diff
@@ -18,7 +18,14 @@ CoreDNS is a general-purpose, authoritative DNS server that provides a backwards
+## We expect your feedback
+
+We are conducting a survey to evaluate the adoption of CoreDNS as the DNS for Kubernetes's cluster. 
+If you are currently using CoreDNS inside a Kubernetes cluster, please, [take 5 minutes to provide us some feedback by filling this survey](https://www.surveymonkey.com/r/SKZQSLK)
```
please, add a period after `..SKZQSLK)`

neolit123 commented:
@fturib so, actually i don't know about the policy here.
the blog is kind of a history item once it goes out, but of course if there are typos or certain details to be changed, we can surely amend.

/assign @kbarnard10 
WDYT?

mistyhacks commented on content/en/blog/_posts/2018-07-10-coredns-ga.md:
```diff
@@ -18,7 +18,14 @@ CoreDNS is a general-purpose, authoritative DNS server that provides a backwards
 In this article, you will learn about the differences in the implementations of kube-dns and CoreDNS, and some of the helpful extensions offered by CoreDNS.
 
-## Implemenation differences
+## We expect your feedback
```
"expect" is not the right word here. Maybe "value" or "appreciate".

fturib commented:
@kbarnard10 : what other place in k8s website, I could inform users about this survey ?

neolit123 commented:
@fturib i think it's still OK to have this in the blog as a retrospect, but let me ask where we can get this announced more broadly.

neolit123 commented:
@fturib i got some ideas where you can get wide exposure for the survey, thanks to Paris (Google):

- kubernetes-users mailing list
https://groups.google.com/forum/#!forum/kubernetes-users
- tweet @ the kubernetes account and they will possibly retweet
https://twitter.com/kubernetesio 
- tweet @ the CNCF account and they will possibly retweet (suggested by Bob Killen)
https://twitter.com/cloudnativefdn
- add to the announcements section of the community meeting
https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit#heading=h.82iqvmivl3kh
(can be added as agenda for the next meeting.)
- kubernetes-users slack channel
- kubernetes sub-reddit:
https://www.reddit.com/r/kubernetes/

if you don't have reddit / twitter etc...the slack / mailing list options are quite good.

fturib commented:
I addressed the comments.

@neolit123  : Thank you for the list ! I appreciate very much.
- kubernetes-users slack channel : I thought of it and added a msg yesterday.

I will go over the other ways asap.

Thanks1

zparnold commented:
/approve

zparnold commented:
/assign @kbarnard10

zacharysarah commented:
/approve
